partiql-types/Cargo.toml

```diff
@@ -26,5 +26,8 @@ ordered-float = "3.*"
 itertools = "0.10.*"
 unicase = "2.6"
 
+miette = { version ="5.*", features = ["fancy"] }
```

Comment:
We should probably upgrade this and the CLI to miette 7.2 to stay aligned with other projects

Reply:
Cool, added the new version.